src/repositories/user.repository.js

```diff
@@ -0,0 +1,76 @@
+import { Op } from 'sequelize';
+import CustomError from '../errors/customError';
+import UserModel from '../models/user';
+
+export default class LocalUserRepository {
```

Comment:
Why didn't you change the name of the class? Maybe `DbUserRepository`?

Reply:
It's a typo. It should be **UserRepository**